src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -734,5 +734,5 @@
     <Folder Include="Linker\Linker\" />
     <Folder Include="utils\" />
   </ItemGroup>
-  <Import Project="..\..\packages\ILRepack.Lib.MSBuild.Task.2.0.15.4\build\ILRepack.Lib.MSBuild.Task.targets" Condition="Exists('..\..\packages\ILRepack.Lib.MSBuild.Task.2.0.15.4\build\ILRepack.Lib.MSBuild.Task.targets')" />
+  <Import Project="ILRepack.targets" Condition="Exists('ILRepack.targets')" />
```

Comment:
Why do we need the `Condition` here. This file *must* exist; if it doesn't, we *should* fail.